packit_service/worker/build/build_helper.py

```diff
@@ -207,9 +240,14 @@ def job_tests(self) -> Optional[JobConfig]:
     @property
     def status_reporter(self) -> StatusReporter:
         if not self._status_reporter:
-            self._status_reporter = StatusReporter(
-                self.project, self.metadata.commit_sha, self.metadata.pr_id
-            )
+            if isinstance(self.project, GitlabProject):
+                self._status_reporter = StatusReporter(
+                    self.base_project, self.metadata.commit_sha, self.metadata.pr_id
+                )
+            else:
+                self._status_reporter = StatusReporter(
+                    self.project, self.metadata.commit_sha, self.metadata.pr_id
+                )
```

Comment:
Maybe this to show the only difference more clearly?

```suggestion
            self._status_reporter = StatusReporter(
                project=self.base_project if isinstance(self.project, GitlabProject) else self.project,
                commit_sha=self.metadata.commit_sha,
                pr_id=self.metadata.pr_id
            )
```

Reply:
Done!